.cursor/skills/types-update/reference.md

```
@@ -0,0 +1,404 @@
+2. Parse the "Potential Conflicts" section for the list of conflicting type names
+3. For each conflicting type, check if it currently appears in the named export block in `src/generated/index.ts`
+4. If it does, remove it from the export block
+5. Parse the "KubeVirt-Only Types" section for new types that might be needed
+6. Check if any new KubeVirt-only types are used by the consumer project (`forklift-console-plugin`). Search for imports from `@forklift-ui/types` that reference KubeVirt type names.
+7. Add any needed new types to the export block
+8. Run `npm run build` to verify
+
+### Auto-fix for Duplicate identifier errors
+
+If `npm run build` fails with errors like:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)